admin/admin-service/src/test/java/io/mosip/admin/controller/ApplicantDetailsControllerTest.java

```diff
@@ -110,12 +105,46 @@ public void getApplicantDetailsWithInvalidRidTest() throws Exception {
     @Test
     @WithUserDetails(value = "global-admin")
     public void testGetRIDDigitalCardSuccess() throws Exception {
-        byte[] pdfbytes="test".getBytes();
-        Mockito.when(applicantDetailService.getRIDDigitalCard(Mockito.anyString(),Mockito.anyBoolean())).thenReturn(pdfbytes);
+        String data="dHN0bWFzIGRzYWttZ2FzIGRma3M=";
+        String str = new String(Files.readAllBytes(Paths.get(getClass().getResource("/responseJson.json").toURI())), StandardCharsets.UTF_8);
+        Mockito.when(restClient.getApi(Mockito.any(), Mockito.any(), Mockito.anyString(), Mockito.any(),
+                                Mockito.any(Class.class))).thenReturn(str);
+        Mockito.when(restClient.getForObject(Mockito.any(),
+                Mockito.any(Class.class))).thenReturn(data);
         mockMvc.perform(MockMvcRequestBuilders.post("/rid-digital-card").param("rid","11234567897").param("isAcknowledged","true")
                         .contentType(MediaType.APPLICATION_JSON_VALUE))
                 .andExpect(status().isOk()).andExpect(MockMvcResultMatchers.content()
                         .contentType(MediaType.APPLICATION_PDF_VALUE));
     }
+    @Test
+    @WithUserDetails(value = "global-admin")
+    public void testGetRIDDigitalCardFailure() throws Exception {
+        AdminDataUtil.checkResponse(
+                mockMvc.perform(MockMvcRequestBuilders.post("/rid-digital-card").param("rid","11234567897").param("isAcknowledged","false"))
+                        .andReturn(),
+                "ADM-AVD-006");
+    }
+    @Test
+    @WithUserDetails(value = "global-admin")
+    public void testGetRIDDigitalCardFailure1() throws Exception {
```

Comment:
Please don't use numbers to differentiate between test case methods. Rename to have some info about what kind of failure.

Reply:
taken care

Reply:
Resolved